src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_ReadPosRankSumTest.java

```diff
@@ -37,9 +38,10 @@
  * </ul>
  *
  */
-//FIXME - this whole class is essentially code duplicated from ReadPosRankSumTest
 public class AS_ReadPosRankSumTest extends AS_RankSumTest implements AS_StandardAnnotation {
 
+    final ReadPosRankSumTest readPosRankSumTest = new ReadPosRankSumTest();
```

Comment:
@droazen I couldn't figure out a way to beat code duplication for `isUsableRead` -- since you can't override `static` methods, every option was a lot more fuss than it was worth.  I made `getElementForRead` delegate to a static method in order to avoid instantiating a `ReadPosRankSumTest`.